konduit-serving-api/src/main/java/ai/konduit/serving/model/PythonConfig.java

```diff
@@ -65,7 +80,140 @@
     @Builder.Default
     private String jobSuffix = "konduit_job";
 
+    public String resolvePythonLibrariesPath() {
+        if(pythonConfigType == null) {
+            log.info("Python config type not specified...");
+            List<CondaDetails> condaInstalls = PythonPathUtils.findCondaInstallations();
```

Comment:
I would only do this if a conda environment type is specified and no environment name is specified. Force the user to be explicit about what they're using.

Reply:
All of the activity is going to be logged to I don't see a big problem here. I was trying to have something where a person has some variables defined and can use the python config without explicitly specifying python paths.

Reply:
Unless he has nothing installed in his environment at all so it's not gonna use javacpp by default and instead prompt the user to use the locally installed python at a custom place.

Reply:
The issue is it assumes conda. Javacpp would be reasonable since it's bundled.